master/internal/rm/kubernetesrm/job.go

```diff
@@ -587,6 +587,16 @@ func (j *job) preparePodUpdateMessage(msgText string) string {
 	return msgText
 }
 
+// PodScheduled checks pod conditions to determine if a pod has been scheduled onto a node.
+func PodScheduled(pod k8sV1.Pod) bool {
```

Comment:
style nit, don't export this function since its usefulness is limited to the k8s package